app/build.gradle

```diff
@@ -95,7 +95,7 @@ dependencies {
     coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.0.10'
 
     // Kotlin
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
+    // implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
```

Comment:
Can you please delete the line instead of commenting it out?

Reply:
Done